components/net/fetch/request.rs

```diff
@@ -4,16 +4,20 @@
 
 use fetch::cors_cache::{BasicCORSCache, CORSCache, CacheRequestDetails};
 use fetch::response::ResponseMethods;
+use http_loader::{NetworkHttpRequestFactory, WrappedHttpResponse};
+use http_loader::{create_http_connector, obtain_response};
```

Comment:
I'm not sure whether it's a good idea for the fetch infrastructure to depend upon `http_loader`.